images/haveged/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM multiarch/debian-debootstrap:armhf-stretch-slim
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+				haveged && \
+	rm -rf /var/lib/apt/lists/*
+
+ENTRYPOINT ["/usr/sbin/haveged", "-F"]
```

Comment:
I am not sure what the default number of bits but maybe add `-w 1024` to be sure.